### PR TITLE
Implement --accept-timeout option on DebuggerRemote and fix bug

### PR DESCRIPTION
**Pull request by ksh8281**
Signed-off-by: Seonghyun Kim <sh8281.kim@samsung.com>

### CODE REVIEW

File: src/debugger/DebuggerTcp.cpp [hunk 1]
```diff
@@ -344,6 +349,28 @@ void DebuggerTcp::init(const char* options, Context* context)
 
     ESCARGOT_LOG_INFO("Waiting for client connection 0.0.0.0:%hd\n", port);
 
+    while (true) {
+        struct pollfd fd[1];
+        fd[0].fd = serverSocket;
+        fd[0].events = POLLIN;
```
Comment (bbrto21) [inline src/debugger/DebuggerTcp.cpp:355]: Why don't you move them out of the loop?

**Review by bbrto21 — APPROVED** [review]
Otherwise, LGTM

File: src/debugger/DebuggerTcp.cpp (hunk 1 above)
Comment (ksh8281) [reply src/debugger/DebuggerTcp.cpp:355]: Good point!

**Review by clover2123 — APPROVED** [review]
LGTM

**Pull request merged**